spec/wor/push/notifications/push_notifications_spec.rb

```diff
@@ -109,6 +115,32 @@
       it 'returns true' do
         expect(send_message).to be true
       end
+
+      context 'when passing a message_content with symbols as keys' do
+        let(:message_content) { { message: 'A message' } }
+
+        it 'sends the message' do
+          expect_any_instance_of(SnsClientMock).to receive(:publish)
+          send_message
+        end
+      end
+
+      context 'when passing a hash with strings as keys' do
+        let(:message_content) { { 'message' => 'A message' } }
+
+        it 'sends the message' do
+          expect_any_instance_of(SnsClientMock).to receive(:publish)
+          send_message
+        end
+      end
+
+      context 'when passing a message_content that is not a Hash' do
```

Comment:
I would take out this case since it's tested then at the push_notifications_validator_spec.rb. This way we have this spec for integration tests and the other one for several cases. What do you think?